server/routes/activities/create-an-activity/handlers/removeEndDate.ts

```diff
@@ -0,0 +1,42 @@
+import { Request, Response } from 'express'
+import { Expose } from 'class-transformer'
+import { IsNotEmpty } from 'class-validator'
+import { ActivityUpdateRequest } from '../../../../@types/activitiesAPI/types'
+import ActivitiesService from '../../../../services/activitiesService'
+
+export class RemoveEndDateOptions {
+  @Expose()
+  @IsNotEmpty({ message: "Please select if you want to change or remove this activity's end date." })
+  removeEndDate: string
+}
+
+export default class RemoveEndDateRoutes {
+  constructor(private readonly activitiesService: ActivitiesService) {}
+
+  GET = async (req: Request, res: Response): Promise<void> => {
+    res.render('pages/activities/create-an-activity/remove-end-date')
+  }
+
+  POST = async (req: Request, res: Response): Promise<void> => {
+    req.session.createJourney.removeEndDate = req.body.removeEndDate
```

Comment:
Not sure where `req.session.createJourney.removeEndDate` is used elsewhere?

Reply:
removed since it isn't being used, and it's not a value we want to keep